package.json

```diff
@@ -37,7 +37,6 @@
   },
   "dependencies": {
     "amd-name-resolver": "0.0.6",
-    "bower": "^1.3.12",
     "bower-config": "^1.3.0",
```

Comment:
Should do the same treatment for the other bower packages (bower-config, bower-endpoint-parser)?

Reply:
Coming from the future.. Was wondering why `ember-cli` still has `bower-config` has dependency.